AWSIoT/AWSIoTDataManager.h

```diff
@@ -416,6 +416,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)enableMetricsCollection:(BOOL)enabled;
 
+/**
+ Set user-specified dictionary of the additional values to be passed as components of
+ connection username.
+
+ *Swift*
+
+ let userMetaData: [String: String] = ["AFRSDK": "ios", "AFRSDKVersion": "1.0.0", "AFRLibVersion":"1.4.1"]
+ iotDataManager.addUserMetaData(userMetaData)
+
+ @param userMetaData A dictionary of key-value meta data pair to be appended to the connection username
```

Comment:
nit: change "meta data pair" to "metadata pairs"

Reply:
Done